src/polymer-project.ts

```diff
@@ -327,6 +327,24 @@ class HtmlSplitter extends Transform {
           this.push(scriptFile);
         }
 
+        for (let i = 0; i < styleTags.length; i++) {
+            let styleTag = styleTags[i];
+            let source = dom5.getTextContent(styleTag);
+            let extension = 'css';
+            let childFilename = `${osPath.basename(filePath)}_style_${i}.${extension}`;
+            let childPath = osPath.join(osPath.dirname(filePath), childFilename);
+            styleTag.childNodes = [];
+            dom5.setAttribute(styleTag, 'src', childFilename);
```

Comment:
A `<link>` tag is definitely what should be here, but there's no support for the `include` property on an external stylesheet. Many styles will still be broken when _split_.